ocs_ci/helpers/dr_helpers.py

```diff
@@ -1628,17 +1633,22 @@ def disable_dr_from_app(secondary_cluster_name):
             cmd = f"oc patch placement {name} -n {namespace}  -p '{params}' --type=json"
             run_cmd(cmd)
 
-    # Delete all drpc
-    run_cmd("oc delete drpc --all -A")
-    sample = TimeoutSampler(
-        timeout=300,
-        sleep=5,
-        func=run_cmd_verify_cli_output,
-        cmd="oc get drpc -A",
-        expected_output_lst="No resources found",
-    )
-    if not sample.wait_for_func_status(result=False):
-        raise Exception("All drpcs are not deleted")
+    time.sleep(60)
```

Comment:
what's the reason for waiting? Would it be possible to replace it with a TimeoutSampler?

Reply:
yes, we can have timeoutsampler